src/main/java/frc/robot/commands/BalanceCommand.java

```diff
@@ -0,0 +1,65 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.util.Functions;
+
+public class BalanceCommand extends CommandBase {
+
+    private Drivetrain m_drive;
+
+    private double startAngle, endPoint;
+
+    private boolean mode;
+ 
+    public BalanceCommand(Drivetrain drive) {
+        m_drive = drive;
+    }
+
+    /*
+     * Determines the values of the vectors of pitch, yaw, and roll. 
+     */
```

Comment:
this is more an in code commnet than a comment for the whole method you can probobly leave the initalize method un-java-documented.